### DISCUSSION TITLE
cannot access docs

### DISCUSSION THREAD

**Discussion by blemmond** [iqtree/iqtree2#237] [Q&A] [answered]
Hi,
Just a general issue, I cannot access any of the documentation (https://www.iqtree.org/doc) pages because of an SSL server issue (e.g. 'SSL_ERROR_UNSUPPORTED_VERSION' on Firefox). I have tried Safari, Firefox, Google Chrome, all latest versions, and get some variation of the same error and cannot navigate around it. I imagine others are having the same issue. I really need to read the documentation to use this tool, so any help is much appreciated! Thanks :)

**Comment by roblanf** [comment, marked as answer]
I have the same issue right now. Luckily the docs are always available on the GitHub wiki here as well:

https://github.com/iqtree/iqtree2/wiki

**Reply by blemmond** [reply]
Thank you!

**Comment by bqminh** [comment]
The error means that the web server doesn't support https protocol. Please change to http://www.iqtree.org
